README.md

```diff
@@ -53,16 +51,20 @@ Change the configuration options by adding arguments to the action:
     # No spaces allowed
 ```
 
-Previous versions of image-actions used `.github/config/image-actions.yml` for configuration. 
-If you're still using that configuration method we suggest that you update it.
+If you would like to modify the defaults, update the `.github/workflows/calibreapp-image-actions.yml` file by adding arguments to the action:
+
+- [jpegQuality](http://sharp.pixelplumbing.com/en/stable/api-output/#jpeg): Number, integer 1-100, default 80 stroed in a string
+- [pngQuality](http://sharp.pixelplumbing.com/en/stable/api-output/#png): Number, integer 1-100, default 80 stored in a string
+- [webpQuality](http://sharp.pixelplumbing.com/en/stable/api-output/#webp): Number, integer 1-100, default 80 stored in a string
+- ignorePaths: a comma seperated string with [globbing](https://www.npmjs.com/package/glob) support of paths to ignore when looking for images to compress
+
+_The `jpegQuality`, `pngQuality` and `webpQuality` config keys will be delivered directly into [sharp’s](http://sharp.pixelplumbing.com) `toFormat` method._
 
-- The above configuration is what `image-actions` uses by default
-- The `jpegQuality`, `pngQuality` and `webpQuality` config keys will be delivered directly into [sharp’s](http://sharp.pixelplumbing.com) `toFormat`. ([JPEG options](http://sharp.pixelplumbing.com/en/stable/api-output/#jpeg), [PNG options](http://sharp.pixelplumbing.com/en/stable/api-output/#png), [Webp options](http://sharp.pixelplumbing.com/en/stable/api-output/#webp))
-- `ignorePaths` allows for path globbing [see the glob package for more details](https://www.npmjs.com/package/glob)
+Previous versions of image-actions used `.github/calibre/image-actions.yml` for configuration. We suggest that you migrate to the newest configuration format by reading the [migration steps](#migration-steps) below.
 
-## Running the action only when images are changed
+### Action Triggers

```

Comment:
I doubt most of our users know what an action trigger is. I'd revert this to the previous title

Reply:
reverted